tests/resources/docker/docker-compose.yml

```diff
@@ -27,7 +27,7 @@ services:
     networks:
       - gateway
       - appwrite
-  
+
```

Comment:
We're using this compose file, only for testing our docker-compose parser. No need to add extra env vars. Actually, it might be better if use a non-appwrite examples here.